unbound-install.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+if [[ "$UID" -ne 0 ]]; then
+	echo "Sorry, you need to run this as root"
+	exit 1
+fi
+
+if [[ -e /etc/debian_version ]]; then
+	VERSION_ID=$(cat /etc/os-release | grep "VERSION_ID")
+	if [[ "$VERSION_ID" = 'VERSION_ID="7"' ]] || [[ "$VERSION_ID" = 'VERSION_ID="8"' ]] || [[ "$VERSION_ID" = 'VERSION_ID="9"' ]]; then
+		OS="debian"
+		elif [[ "$VERSION_ID" = 'VERSION_ID="12.04"' ]] || [[ "$VERSION_ID" = 'VERSION_ID="14.04"' ]] || [[ "$VERSION_ID" = 'VERSION_ID="16.04"' ]] || [[ "$VERSION_ID" = 'VERSION_ID="16.10"' ]] || [[ "$VERSION_ID" = 'VERSION_ID="17.04"' ]]; then
+			OS="ubuntu"
+			else
+				echo "$VERSION_ID not supported"
+				while [[ $CONTINUE != "y" && $CONTINUE != "n" ]]; do
+					read -p "Continue anyway? [y/n]: " -e CONTINUE
+				done
+		if [[ "$CONTINUE" = "n" ]]; then
+			echo "Ok, bye !"
+			exit 4
+		fi
+	fi	
+elif [[ -e /etc/centos-release || -e /etc/redhat-release || -e /etc/system-release && ! -e /etc/fedora-release ]]; then
```

Comment:
`/etc/fedora-release` is not needed here, is it?

Reply:
Fedora uses dnf but you're right: it's not needed as yum redirects to dnf for what I know.

Reply:
I was pointing this out because of line 28 😄

Reply:
I'm still learning bash, so for OS detection I used your functions in the openvpn-install.sh.

Reply:
Sorry I didn't see the `!`.... Nevermind